container.go

```diff
@@ -51,6 +62,8 @@ type ContainerRequest struct {
 	WaitingFor   wait.Strategy
 	Name         string // for specifying container name
 	Privileged   bool   // for starting privileged container
+	Entrypoint   []string
+	DontRemove   bool
```

Comment:
There is another PR that has the same implementation, can you remove this part as well? Thanks